css/style.css

```diff
@@ -105,94 +159,61 @@ video {
   width: 14px;
   height: 14px;
 }
-.pointer-list__oval--gray{
+.pointer-list__oval--gray {
   background-color: rgba(51, 51, 51, 0.3);
 }
-.pointer-list__oval--black{
+.pointer-list__oval--black {
   background-color: #333333;
 }
-@media screen and (max-width: 960px){
-  .pointer-list__oval{
-    margin-left: 11px;
-    margin-right: 11px;
-    width: 10px;
-    height: 10px;
-  }
-}
 
- /*end comodities*/
+/* End carrousel*/
 
-/* We should use a mobile first approach having out breack point at 768px */
-.button {
-	font-family: roboto, sans-serif;
-	font-weight: 500;
-	border-radius: 2px;
-	overflow: hidden;
-}
-
-.button--small {
-	width: 75px;
-	height: 30px;
-	font-size: 12px;
-}
-
-.button--medium {
-	width: 120px;
-	height: 45px;
-	font-size: 18px;
-}
-
-.button--large {
-	width: 160px;
-	height: 60px;
-	font-size: 22px;
-}
-
-.button--full {
-	width: 100%;
-}
-
-.button--primary {
-	background-color: #29b6f6;
-}
-
-.button--primary-light {
-	background-color: #73e8ff;
-}
-
-.button--primary-dark {
-	background-color: #0086c3;
-}
+/* For desktop */
+@media only screen and (min-width: 960px) {
+  
+  .button {
+    min-width: 320px;
+    height: 60px;
+    font-size: 22px;
+    -webkit-transition-duration: 0.4s;
+    transition-duration: 0.4s;
+  }
 
-.button--secondary {
-	color: #ffffff;
-	background-color: #1565c0;
-}
+  .button--color--primary:hover {
+    background-color: #0086c3;
+  }
+  
+  .button--color--primary-dark:hover {
+    background-color: #29b6f6;
+  }
 
-.button--secondary-light {
```

Comment:
this is the correct naming convention for the secondary color of the button.
please check your classes names.